index.md

```diff
@@ -1,125 +1,93 @@
 ---
-layout: default
+layout: splash
+classes: landing
+
 ---
+<link rel="stylesheet" type="text/css" href="_includes/content-scroller.css"/>
+<link rel="stylesheet" type="text/css" href="_includes/styles.css"/>
+
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script type="text/javascript" src="_includes/config.js"></script>
+<script type="text/javascript" src="_includes/populate-content.js"></script>
+
+# Azure AI Gallery
+
+<div id="datadump">
+
+</div>
+
 
+### The AI Gallery enables a growing community of developers and data scientists to browse curated AI examples, including:
+<br/>
 
-# What is the AI Gallery
-The AI Gallery enables a growing community of developers and data scientists to browse curated AI examples, including:
-1. ML models 
+1. ML models
 2. ML experiments
 3. Example use cases for ML models in the real world
 
-# Featured Models
-<div class="github-card" data-github="onnx/models" data-width="400" data-height="" data-theme="default"></div>
-<script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>
+<br/>
+<br/>
+* * *
+# Featured
 
-# Featured Examples
-We can prettify this, or just point to https://github.com/search?q=machinelearningsamples (none of these repos are properly tagged with topics...)
 
-# Top GitHub Content (Discover Experience)
-## Topic Search: 
-<input type='text' id='keyword' name='keyword' value='mnist'>
-<input type="button" id="btnSearch" value="Search"/>
-<div class="github-widget" data-type="search"></div>
+## Models
```

Comment:
UX question (probably for Jordan), should we add some specification here to set the expectation that models' actionable items are to try/deploy, and examples's actionable items are to follow the repo's readme(?)

Reply:
@jpe316